distod/src/main/scala/com/github/codelionx/distod/actors/partitionMgmt/PartitionManager.scala

```diff
@@ -107,175 +115,172 @@ class PartitionManager(
       )
       partitions = CompactingPartitionMap(compactionSettings).from(singletonPartitions)
       stash.unstashAll(
-        behavior(attributes, PendingJobMap.empty)
+        behavior(attributes)
       )
     } else {
       initialize(attributes, singletonPartitions)
     }
 
-  private def behavior(
-      attributes: Seq[Int],
-      pendingJobs: PendingJobMap[CandidateSet, PendingResponse]
-  ): Behavior[PartitionCommand] = {
-    def next(
-        _attributes: Seq[Int] = attributes,
-        _pendingJobs: PendingJobMap[CandidateSet, PendingResponse] = pendingJobs,
-    ): Behavior[PartitionCommand] =
-      behavior(_attributes, _pendingJobs)
-
-    Behaviors.receiveMessage {
-
-      // inserts
-      case SetAttributes(ignored) =>
-        context.log.warn("Attempt to change attributes to {} after initialization step, ignoring!", ignored)
-        Behaviors.same
-
-      case InsertPartition(key, _: FullPartition) if key.size == 1 =>
-        context.log.warn("Attempt to change initial partitions (key.size == 1) after initialization step, ignoring!")
-        Behaviors.same
-
-      case InsertPartition(key, value: StrippedPartition) =>
-        context.log.debug("Inserting partition for key {}", key)
-        partitions + (key -> value)
-        Behaviors.same
-
-      // request attributes
-      case LookupAttributes(replyTo) =>
-        replyTo ! AttributesFound(attributes)
-        Behaviors.same
-
-      // request from a partition replicator
-      case OpenConnection(replyTo) =>
-        context.log.info("Opening sidechannel to {}", replyTo)
-        context.spawn(PartitionManagerEndpoint(partitions, replyTo, attributes), PartitionManagerEndpoint.name())
-        Behaviors.same
-
-      // single-attr keys
-      case LookupError(key, replyTo) if key.size == 1 =>
-        partitions.getSingletonPartition(key) match {
-          case Some(p) =>
-            replyTo ! ErrorFound(key, p.stripped.error)
-            Behaviors.same
-          case None =>
-            throw new IllegalAccessException(s"Full partition for key ${key} not found! Can not compute error.")
-        }
+  private def behavior(attributes: Seq[Int]): Behavior[PartitionCommand] = Behaviors.receiveMessage {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Method is longer than 50 lines](https://app.codacy.com/manual/CodeLionX/distod/pullRequest?prid=5048550)